src/components/NavSection.vue

```diff
@@ -1,42 +1,52 @@
 <template>
-  <nav :aria-label="$t('header.aria.primary')" class="navbar">
-    <div class="navbar-brand text-white">
-      <!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
-      <NuxtLink class="logo" :to="localePath('/')">
-        <img
-          alt="Openverse logo mark"
-          src="~/assets/logo.svg?data"
-          style="padding-right: 24px"
+  <!-- Refer to the Bulma docs for markup: https://bulma.io/documentation/components/navbar/ -->
+  <nav
+    role="navigation"
+    class="navbar embedded"
+    :aria-label="$t('header.aria.primary')"
+  >
+    <div class="navbar-brand flex-grow md:flex-grow-0">
+      <NuxtLink
+        to="/"
+        class="navbar-item"
+        style="align-self: center; line-height: 0"
+      >
+        <!-- width and height chosen w.r.t. viewBox "0 0 280 42" -->
+        <OpenverseLogo
+          :style="{ width: '160px', height: '24px' }"
           width="160"
           height="24"
+          alt="Openverse logo mark"
```

Comment:
"mark" seems redundant with `logo` to me.

```suggestion
          alt="Openverse logo"
```

Reply:
Fixed in 53058d1.